api/api.go

```diff
@@ -480,7 +480,7 @@ func getOrganization(ctx context.Context, logger *logger.Logger, organizationNam
 	err = graphQLclient.Run(ctx, request, &response)
 
 	if err != nil || response.Organization.ID == "" {
-		err = fmt.Errorf("Unable to find organization %s of vcs-type %s", organizationName, organizationVcs)
+		err = errors.Wrap(err, fmt.Sprintf("Unable to find organization %s of vcs-type %s", organizationName, organizationVcs))
```

Comment:
You can use `errors.Wrapf(err, "...", orgName, orgVCS)` , no need for fmt.Sprintf

Reply:
Thanks @dnephin , I added this in a separate small PR here:

https://github.com/CircleCI-Public/circleci-cli/pull/137